src/store/modules/facility/actions.ts

```diff
@@ -141,6 +141,33 @@ const actions: ActionTree<FacilityState, RootState> = {
       facilityTypeId: ''
     })
     commit(types.FACILITY_LIST_UPDATED , { facilities: [], total: 0 });
+  },
+
+  async getFacilityProductStores({ commit, state }, params) {
+    let productStores = []
+    const payload = {
+      inputFields: {
+        facilityId: params.facilityId
+      },
+      viewSize: 100,
+      entityName: 'ProductStoreFacility',
+      filterByDate: 'Y',
+      fieldList: ['fromDate', 'productStoreId']
+    }
+
+    try {
+      const resp = await FacilityService.getFacilityProductStores(payload)
+
+      if (!hasError(resp) || resp.data.error === 'No record found') {
+        productStores = resp.data.docs ? resp.data.docs : []
```

Comment:
No need to use ternary, as we will already check whether the resp has some data or not, using count param